README.md

```diff
@@ -7,23 +7,55 @@ Cyclone DDS is ready to use. It seeks to give the fastest, easiest, and most rob
 
 1. Install:
 
-  ```
-  apt install ros-eloquent-rmw-cyclonedds-cpp
-  ```
-  or
-  ```
-  apt install ros-dashing-rmw-cyclonedds-cpp
-  ```
+   ```
+   apt install ros-eloquent-rmw-cyclonedds-cpp
+   ```
+   or
+   ```
+   apt install ros-dashing-rmw-cyclonedds-cpp
+   ```
 
-2) Set env variable and run ROS2 apps as usual:
+2. Set env variable and run ROS2 apps as usual:
 
-  ```export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp```
+   ```export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp```
 
-3) Confirm RMW: In Eloquent, to confirm which RMW you're using:
+3. Confirm RMW: In Eloquent, to confirm which RMW you're using:
 
    ```ros2 doctor --report```
 
 
+## Performance recommendations
+
+With large samples (100s of kilobytes), excessive latency can be caused by running out of space in the OS-level receive buffer. For this reason, on Linux, we recommend increasing the buffer size:
+* Temporarily (until reboot): `sudo sysctl -w net.core.rmem_max=8388608`
+* Permanently: `echo "net.core.rmem_max=8388608\n" | sudo tee /etc/sysctl.d/60-cyclonedds.conf`
```

Comment:
The kernel has a default and a maximum. This only raises the maximum, which means cyclone will still get the default, then check if it is < 1MB and if so ask for 1MB.

If you want to actually get more than 1MB, you either have to raise the kernel default (in which cyclone will observe it is > 1MB and leave it alone) or configure cyclone to ask for more. That's what Internal/MinimumSocketReceiveBufferSize size is for.

Reply:
That's a bit confusing.... I have an idea.

Reply:
My Idea: https://github.com/eclipse-cyclonedds/cyclonedds/pull/489